libcxx/docs/Status/Cxx20Papers.csv

```diff
@@ -152,7 +152,7 @@
 "`P1855R0 <https://wg21.link/P1855R0>`__","Make ``<compare>``\  freestanding","2019-11 (Belfast)","","",""
 "`P1862R1 <https://wg21.link/P1862R1>`__","Ranges adaptors for non-copyable iterators","2019-11 (Belfast)","|Complete|","16.0",""
 "`P1865R1 <https://wg21.link/P1865R1>`__","Add max() to latch and barrier","2019-11 (Belfast)","|Complete|","11.0",""
-"`P1869R1 <https://wg21.link/P1869R1>`__","Rename 'condition_variable_any' interruptible wait methods","2019-11 (Belfast)","","",""
+"`P1869R1 <https://wg21.link/P1869R1>`__","Rename 'condition_variable_any' interruptible wait methods","2019-11 (Belfast)","|Complete|","18.0","The paper is implemented but the features are experimental and can be enabled via ``-fexperimental-library``."
```

Comment:
We don't say anything about `-fexperimental-library` anywhere else. This is also way too much text for a table IMO.

Reply:
I copied the note for consistency from here (`P0660R10 <https://wg21.link/P0660R10>`__ | Stop Token and Joining Thread, Rev 10 ):
https://github.com/H-G-Hristov/llvm-project/blob/1bbbb628cca7a80d1262ffc5db51ef4064c5c5ea/libcxx/docs/Status/Cxx20Papers.csv#L108

Should I cut the message short in both places for consistency, e.g.: 

>The paper is implemented but the features are experimental.

or keep them as is? I guess the original note was a footnote.